corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -97,15 +106,18 @@ def _get_template_translation_el(self):
         #
         # Currently operating under the assumption that every xForm has at least
         # one translation element, that each translation element has a text node
-        # for each question and that each text node has a value node under it
-        template_translation_el = None
-        # Get a translation element to be used as a template for new elements
-        for lang in self.langs:
+        # for each question and that each text node has a value node under it.
+        # Get a translation element to be used as a template for new elements, preferably of default lang
+        default_lang = self.app.default_language
+        default_trans_el = self.itext.find("./{f}translation[@lang='%s']" % default_lang)
+        if default_trans_el.exists():
+            return default_trans_el
+        all_langs = self.app.langs
+        for lang in all_langs.remove(default_lang):
```

Comment:
This will throw `TypeError` because `all_langs.remove(default_lang)` returns None, not the new value of `all_langs`.

You'll need to do the `remove()` before the `for` loop, or just skip `default_lang`:
```python
    for lang in all_langs:
        if lang == default_lang:
            continue
        # ... etc. ...
```

Reply:
updated.